src/families/ethereum/cli-transaction.js

```diff
@@ -74,7 +74,7 @@ function inferTransactions(
       userGasLimit: new BigNumber(opts.gasLimit),
       estimatedGasLimit: null
     };
-  });
+  }).map();
```

Comment:
Pretty obscure.

`undefined is not a function` no?

if there is magic behind that, well, please let's just explicitly lodash.`flatMap`.